todo_app/data/trello_items.py

```diff
@@ -0,0 +1,111 @@
+import json
+from os import getenv
+from flask import session
+import requests
+
+from todo_app.domain.item import Item
+
+# use env var for prefix
+cards_url = "https://api.trello.com/1/cards"
+board_url = "https://api.trello.com/1/boards/65c4fed365273e251af75aa5"
+todo_list_id = "65c4fed42c4a7016163343f9"
```

Comment:
I can see you've dealt with the first of these in the next exercise, but it would definitely be nice to also pull the `todo_list_id` out into an environment variable - otherwise all of our environments are stuck using the same list!